tests/integration/ambient/bookinfo_test.go

```diff
@@ -316,31 +319,72 @@ func applyDefaultRouting(t framework.TestContext, nsConfig namespace.Instance) {
 	applyFileOrFail(t, nsConfig.Name(), routingV1)
 }
 
-func setupWaypoints(t framework.TestContext, nsConfig namespace.Instance, sa string) {
+func applyWaypoint(t framework.TestContext, ns namespace.Instance) {
 	istioctl.NewOrFail(t, t, istioctl.Config{}).InvokeOrFail(t, []string{
 		"x",
 		"waypoint",
 		"apply",
 		"--namespace",
-		nsConfig.Name(),
-		"--service-account",
-		sa,
+		ns.Name(),
 		"--wait",
 	})
 }
 
-func deleteWaypoints(t framework.TestContext, nsConfig namespace.Instance, sa string) {
+func setupWaypoints(t framework.TestContext, nsConfig namespace.Instance, service string) {
```

Comment:
I mentioned this in https://github.com/istio/istio/pull/49353

..but I think these setup teardown funcs probably actually belong in `pkg/test/framework/components/ambient/waypoint.go`?

We should have generalized funcs anyway, and they shouldn't belong to `bookinfo` proper, since we want to get rid of `bookinfo` tests

Reply:
that's a good point because they're used by the gateway conformance tests too so they're already not bookinfo-specific